infra/terraform/templates/ecs.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_ecs_cluster" "wellcomecollection" {
+  name = "wellcomecollection"
+}
+
+resource "aws_ecs_task_definition" "wellcomecollection" {
+  family = "wellcomecollection"
+  container_definitions = "${file("container-definitions.json")}"
+}
+
+resource "aws_ecs_service" "wellcomecollection" {
+  name = "wellcomecollection"
+  cluster = "${aws_ecs_cluster.wellcomecollection.id}"
+  task_definition = "${aws_ecs_task_definition.wellcomecollection.arn}"
+  depends_on = ["aws_iam_role_policy.ecs_service_policy"]
+  desired_count = 1
```

Comment:
What happens to speed of deployment if this number is higher. Does is replace containers sequentially?